src/templates/playgroundTemplate.ts

```diff
@@ -1,12 +1,15 @@
-const template: string = `//select the database to use.
+const template: string = `// Select the database to use
 use('test');
-//run a find command.
-db.my_collection.find({foo: 'bar'});
-//run an aggregation.
+
+// Run a find command
+db.myCollection.find({ foo: 'bar' });
+
+// Run an aggregation
 const agg = [
-  {$match: {foo: 'bar'}}
+  { $match: { foo: 'bar' } }
 ];
-db.my_collection.aggregate(agg);
+
+db.myCollection.aggregate(agg);
 `;
 
```

Comment:
@Anemy had a preference for closing the sentences in the comments with a `.`. I think it makes sense, although I don't have a strong opinion either way.

Reply:
I don't have a strong opinion either, I prefer without, but I will return them back if having them makes sense for both of you.